code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -705,15 +727,17 @@
 					if(isliving(uneq_piece.loc))
 						var/mob/living/L = uneq_piece.loc
 						L.unEquip(uneq_piece, 1)
-					uneq_piece.flags &= ~NODROP
+					//uneq_piece.flags &= ~NODROP
```

Comment:
Please don't comment out code that is being removed- Especially when it has the same comment copy-pasted after it over 5 times. Just delete the line and leave it clean.